### ISSUE TITLE
Open a reference's file with a custom program

### ISSUE THREAD

**Issue by Djoop** [rafaqz/citation.vim#30]
Hi, I'm using a lot citation.vim to open the articles (the actual pdf files). Following the readme I'm using
```
nnoremap <silent>[unite]co :<C-u>Unite -input=<C-R><C-W> -default-action=start -force-immediately citation/file<cr>
```
It works well but the files are opened with the default pdf viewer. I do use 2 different pdf viewers depending on whether I want to read or annotate the articles, so how could I adapt the previous command to open the article with a custom program? The fact that it goes through unite makes it difficult for me to understand how the command above can be adapted… maybe it's actually more a question related to unite than this plugin, but I'm not sure.

**Comment by rafaqz** [comment]
Yeah unite. But then, its still just your system tool. On Linux I think its `xgd-open`. You can set your defaults in these tools.

**Comment by Djoop** [comment]
Thanks. But so what if I don't want to change the default but really define a new shortcut… is there a way to just fetch the path to the file and just use it in any custom command? Or the only way is via citation/file, and thus really via unite? Well if it is the only way, I'll double check unite documentation and this can be closed.

**Comment by rafaqz** [comment]
Im not sure, check the unite docs.
